vignettes/eisaR.Rmd

```diff
@@ -111,9 +111,6 @@ We next align the reads to a mini-genome (fasta file `extdata/hg19sub.fa`) using
 `qAlign`:  
 
 ```{r align}
-sampleFile <- "extdata/samples_chip_single.txt"
-genomeFile <- "extdata/hg19sub.fa"
-
 proj <- qAlign("extdata/samples_rna_single.txt", "extdata/hg19sub.fa",
```

Comment:
If at all, you could specify the params name for clarity (`sampleFile` and `genome`)?

Reply:
Yes, good point. In fact that would be even better (specifying the parameter names).

Reply:
Regarding `featureCounts` as an alternative to `qAlign` and `qCount`: Would that also cover alignments? We think this part of the vignette should help people with little experience in RNA-seq data analysis, so ideally covering the whole workflow. But I'll be happy to consider a PR to mention alternatives.

Reply:
Re: changes - sure!
Re: featureCounts - that would be an alternative to `qCount` only, I always appreciated the blazing fast speed of it while keeping the quants accurate!
I'm traveling these days but once the PR is ready, happy to do so ;)